### ISSUE TITLE
Report an error in a larger vault

### ISSUE THREAD

**Issue by KuiyueRO** [aqav/obsidian-canvas-links#1]
![image](https://user-images.githubusercontent.com/63145205/233761393-19c3ad19-34c2-447f-a66a-bb1fee113bfe.png)

```
Uncaught (in promise) TypeError: nodes is not iterable
    at CanvasView.getCanvas (plugin:obsidian-canvas-view:96:26)
```

**Comment by aqav** [comment]
Hi @KuiyueRO It will be helpful if you can provide more info, like the canvas file you has meet the error(use text editor to open it).

**Comment by aqav** [comment]
Hi @KuiyueRO I guess you have empty canvas in your vault. I have fix that 9ed7e99, plz do the follow to apply patch:

1. git clone this repo
2. npm install
3. replace the main.js in your Plugins folder

**Comment by KuiyueRO** [comment]
It works great! ! Thanks a lot for your fix.
